app/scripts/modules/core/src/task/monitor/taskMonitorService.js

```diff
@@ -1,86 +0,0 @@
-'use strict';
-const angular = require('angular');
-
-import _ from 'lodash';
-
-import { TASK_READ_SERVICE } from 'core';
-import { TASKS_MONITOR_DIRECTIVE } from './taskMonitor.directive';
-
-module.exports = angular
-  .module('spinnaker.tasks.monitor.service', [TASKS_MONITOR_DIRECTIVE, TASK_READ_SERVICE])
-  .factory('taskMonitorService', function($log, taskReader, $timeout) {
-    function buildTaskMonitor(params) {
-      var monitor = {
-        submitting: false,
-        task: null,
-        error: false,
-        errorMessage: null,
-        title: params.title,
-        application: params.application,
-        onTaskComplete: params.onTaskComplete || angular.noop,
-        modalInstance: params.modalInstance,
-        monitorInterval: params.monitorInterval || 1000,
-        submitMethod: params.submitMethod,
-      };
-
-      monitor.onModalClose = function() {
-        if (monitor.task && monitor.task.poller) {
-          $timeout.cancel(monitor.task.poller);
-        }
-      };
-
-      monitor.modalInstance.result.then(monitor.onModalClose, monitor.onModalClose);
-
-      monitor.closeModal = function() {
-        try {
-          monitor.modalInstance.dismiss();
-        } catch (e) {
-          // modal was already closed
-        }
-      };
-
-      monitor.startSubmit = function() {
-        monitor.submitting = true;
-        monitor.task = null;
-        monitor.error = false;
-        monitor.errorMessage = null;
-      };
-
-      monitor.setError = function(task) {
-        if (task) {
-          monitor.task = task;
-          monitor.errorMessage = monitor.task.failureMessage || 'There was an unknown server error.';
```

Comment:
@anotherchrisberry where did all this code go?

Reply:
Dead code - it hasn't been used anywhere since...early last year, I think?